crates/turborepo-api-client/src/lib.rs

```diff
@@ -463,6 +464,50 @@ impl APIClient {
             use_preflight,
         })
     }
+
+    /// Create a new request builder with the preflight check done,
+    /// team parameters added, and CI header.
```

Comment:
Probably worth mentioning that this sets the content type to json
```suggestion
    /// team parameters added, CI header, and a content type of json.
```